mappings/net/minecraft/util/collection/ExpandingLongLinkedOpenHashSet.mapping

```diff
@@ -0,0 +1,55 @@
+CLASS net/minecraft/class_6136 net/minecraft/util/collection/ExpandingLongLinkedOpenHashSet
+	COMMENT A custom implementation of LongLinkedOpenHashSet with an unknown final size.
+	COMMENT <p>
+	COMMENT Uses a Long2LongOpenLinkedHashMap that is resized on write,
+	COMMENT allowing it to increase its capacity if needed.
+	COMMENT It also attempts to "pack" multiple values into one index.
+	FIELD field_31715 buffer Lnet/minecraft/class_6136$class_6137;
+	METHOD <init> (IF)V
+		ARG 1 expectedSize
+		ARG 2 loadFactor
+	METHOD add (J)Z
+		ARG 1 value
+	METHOD rem (J)Z
+		ARG 1 value
+	METHOD size ()I
+		COMMENT @throws UnsupportedOperationException
+	CLASS class_6137 Buffer
+		FIELD field_31716 STARTING_OFFSET I
+		FIELD field_31717 MIDDLE_OFFSET I
+		FIELD field_31718 BIT_OFFSET_DIFFERENCE I
+		FIELD field_31719 MIDDLE_BIT_OFFSET I
+		FIELD field_31720 LOWER_BIT_OFFSET I
+		FIELD field_31721 HIGHER_BIT_OFFSET I
+		FIELD field_31722 TOTAL_BIT_MASK J
+		FIELD field_31723 lasWrittenIndex I
+		FIELD field_31724 lasWrittenKey J
```

Comment:
```suggestion
		FIELD field_31723 lastIndex I
		FIELD field_31724 lastKey J
```
methink those two are updated on reads, aren't they? Also use `last` than `las` prefix

Reply:
Keys and indexes are different. The parent class has an array of keys and an array of values, and here `lastWrittenKey = this.keys[index]` and `value = this.values[lastWrittenIndex]`

Reply:
> methink those two are updated on reads, aren't they?
No. They're updated when adding/removing. In fact there aren't any methods that read without a corresponding write.

The only operations we have here are adds and removes, with one of them performing a pop (read and remove).

Reply:
Why is `Written` needed here? It isn't crucial to the nature of the field and just makes the name longer. I think just `last` prefix suffices.